dockerfiles/keycloak/kc_realm_user.sh

```diff
@@ -33,10 +33,27 @@ if [ $KEYCLOAK_USER ] && [ $KEYCLOAK_PASSWORD ]; then
     /opt/jboss/keycloak/bin/add-user-keycloak.sh --user $KEYCLOAK_USER --password $KEYCLOAK_PASSWORD
 fi
```

Comment:
Do we need to port these changes to downstream to keep files in sync @tolusha @nickboldt?

Reply:
@skabashnyuk 
no, `kc_realm_user.sh ` is executed when Eclipse Che is deployed by helm
For other cases we do everything on operator side 
https://github.com/eclipse/che-operator/blob/master/pkg/deploy/identity-provider/deployment_keycloak.go#L154-L157